lib/Makefile.am

```diff
@@ -77,3 +77,13 @@ tests_testRFC1738_LDADD= \
 tests_testRFC1738_LDFLAGS = $(LIBADD_DL)
 
 testHeaders: $(top_srcdir)/include/*.h
+
+if ENABLE_FUZZ_DECODE
+EXTRA_PROGRAMS = fuzzRFC1738
+fuzzRFC1738_SOURCES = tests/fuzzRFC1738.cc
```

Comment:
I do not see any benefit here. This function has a deterministic mapping of inputA to outputB. While we are currently lacking a test of the mapping, a fuzzer is the least efficient way to test.

Reply:
I agree with that.